index.html

```diff
@@ -668,6 +681,110 @@ <h2 id="constraints">Unconstrained Display Surface Selection</h2>
     </section>
 
 
+    <section>
+      <h2 id="changing-display-surfaces">Changing Display Surfaces</h2>
+
+
+      <p>
+        The user agent MUST NOT change the [=display surface=]
+        associated with a [=capture-session=] unless the user has
+        explicitly indicated that they want this change to be
+        performed by interacting with user agent or operating system.
+
+        If the user indicates a change of [=display surface=], the
+        user agent MUST run the following steps:
+      </p>
+
+
+      <ol>
+        <li>
+          <p>If the [=capture-session=] does not have an associated
+            {{CaptureController}}, <var>controller</var>, abort these steps.</p>
+        </li>
+
+
+        <li>
+          <p>If <var>controller.{{CaptureController/[[DisplaySurfaceChangeCallback]]}}</var>
+            is null, abort these steps.</p>
+        </li>
+
+
+        <li>
+          <p>Stop delivering frames from all sources in the
+            [=capture-source-set=] associated with the [=capture-session=].
+          </p>
+        </li>
+
+
+        <li>
+          <p>Let <var>newSources</var> be the set of sources selected by the
+            user when the user chose a new [=display surface=].</p>
+
+
+          <p>
+            <var>newSources</var> MUST fulfill the same requirements as the
+            [=capture-source-set=] formed when {{MediaDevices/getDisplayMedia}}
+            is called.</p>
+        </li>
+
+
+        <li>
+          <p>Queue a task to:</p>
```

Comment:
Will it have a link to the definition?
We should probably start using a task source as well.